src/components/SessionStore/SessionStore.component.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ * Class that extends the koa-seesion2 Store object and serves as the binding layer to the session storage
+ */
+
+import {Store} from 'koa-session2';
+import {PersistenSessionStorage} from '../../models/session.persistent.storage.model';
+import {MemorySessionStorage} from '../../models/session.memory.storage.model';
+import {log} from '../../utils/logging.util';
+
+export default class SessionStore extends Store {
+  /**
+   * Initises the SessionStore and the underlying Storage component.
+   * If the memory paramter is used one should be aware that sharring of sessions across instances of the application
+   * is not pissoble. The feaure is intended for testing only.
```

Comment:
**If the memory parameter is used one should be aware that sharing of sessions across instances of the application
* is not possible. The feature is intended for testing only.**